bot.py

```diff
@@ -5,25 +5,82 @@
 import random
 from databasecontroller import DatabaseController
 from discord.ext import commands
+import discord
 
+botPrefix = "-picker" # TODO: Get a real prefix??
 db = DatabaseController()
 
 load_dotenv()
 TOKEN = os.getenv('DISCORD_TOKEN')
 
-bot = commands.Bot(command_prefix='?')
-
+bot = commands.Bot(botPrefix)
 
 @bot.event
 async def on_ready():
-    print(f"{bot.user.name} has connected to Discord!")
+    print(f'{bot.user.name} has connected to Discord!')
+
 
 
 @bot.event
 async def on_message(message):
     # TODO: Make this more robust. I'm just trolilng with this
-    id = message.author.id
-    print(id)
+    if message.content.startswith(botPrefix):
+        botReplies = processMessages(message)
+
+        for reply in botReplies:
+            await message.reply(reply)
+
+def processMessages(message):
```

Comment:
for the newlist thing, usage is like:

-picker newlist {listname}

ex.
-picker newlist booba